reference/forms/types/password.rst

```diff
@@ -76,4 +76,10 @@ The default value is ``''`` (the empty string).
 
 .. include:: /reference/forms/types/options/required.rst.inc
 
-.. include:: /reference/forms/types/options/trim.rst.inc
+trim
+~~~~
+**type**: ``Boolean`` **default**: ``false``
+If true, the whitespace of the submitted string value will be stripped
+via the ``trim()`` function when the data is bound. This guarantees that
+if a value is submitted with extra whitespace, it will be removed before
+the value is merged back onto the underlying object.
```

Comment:
I duplicated the text from the underlying object because the default trim option is "true" for the different form fields (except for the password field type where its overloaded to "false").